processing/tests/conftest.py

```diff
@@ -67,24 +74,43 @@ def files_data_for_pipeline(files_data):
     return files_data
 
 
+alembic_cfg = Config("alembic.ini")
+
+
+@pytest.fixture(scope="session")
+def use_temp_env_var():
+    with patch.dict(os.environ, {"USE_TEST_DB": "1"}):
+        yield
+
+
 @pytest.fixture
-def setup_database():
-    test_db_name = uuid.uuid4().hex
-    db_url = f"sqlite:///./{test_db_name}.db"
-    engine = create_engine(db_url, connect_args={"check_same_thread": False})
-    Base.metadata.create_all(bind=engine)
-    session_local = sessionmaker(
-        autocommit=False, autoflush=False, bind=engine
-    )
-    session = session_local()
-    yield session
-    session.close()
-    Base.metadata.drop_all(bind=engine)
-    remove_db(test_db_name)
+def test_db_engine():
```

Comment:
May you rename it to `testing_db_engine` or `db_test_engine` so we won't confuse it with tests

Reply:
agreed - renamed to db_test_engine